backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,234 @@
+                        text=text,
+                        content_object=project,
+                        embedding=embedding,
+                        save=False,
+                    )
+                )
+            ]
+        except openai.OpenAIError as e:
+            self.stdout.write(self.style.ERROR(f"OpenAI API error for project {project.key}: {e}"))
+            return []
+
+    def extract_project_content(self, project: Project) -> tuple[str, str]:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)